workflows/diagnostics/fv3net/diagnostics/prognostic_run/views/matplotlib.py

```diff
@@ -143,39 +195,38 @@ def plot_cubed_sphere_map(
         metrics_for_title = {}
 
     variables_to_plot = run_diags.matching_variables(varfilter)
+    job_args = []
+    if "lonb" and "latb" in run_diags.variables:
+        coord_vars = ["lon", "lat", "lonb", "latb"]
+    else:
+        coord_vars = ["lon", "lat"]
     for varname in variables_to_plot:
         vmin, vmax, cmap = _get_cmap_kwargs(run_diags, varname, robust=True)
         for run in run_diags.runs:
-            logging.info(f"plotting {varname} in {run}")
-            shortname = varname.split(varfilter)[0][:-1]
-            if "lonb" and "latb" in run_diags.variables:
-                COORD_VARS = ["lon", "lat", "lonb", "latb"]
-            else:
-                COORD_VARS = ["lon", "lat"]
-            ds = run_diags.get_variables(run, COORD_VARS + [varname])
-            plot_title = _render_map_title(
-                run_metrics, shortname, run, metrics_for_title
-            )
-            fig, ax = plt.subplots(
-                figsize=(6, 3), subplot_kw={"projection": ccrs.Robinson()}
-            )
-            if "ncol" in ds.sizes:
-                grid_metadata = GridMetadataScream("ncol", "lon", "lat")
-                fv3viz.plot_cube(
+            ds = run_diags.get_variables(run, coord_vars + [varname])
+            job_args.append(
+                (
                     ds,
+                    run,
                     varname,
-                    grid_metadata=grid_metadata,
-                    ax=ax,
-                    vmin=vmin,
-                    vmax=vmax,
-                    cmap=cmap,
+                    vmin,
+                    vmax,
+                    cmap,
+                    metrics_for_title,
+                    run_diags,
+                    run_metrics,
+                    varfilter,
                 )
-            else:
-                fv3viz.plot_cube(ds, varname, ax=ax, vmin=vmin, vmax=vmax, cmap=cmap)
-            ax.set_title(plot_title)
-            plt.subplots_adjust(left=0.01, right=0.75, bottom=0.02)
-            data[varname][run] = MatplotlibFigure(fig, width="500px")
-            plt.close(fig)
+            )
+
+    jobs = [delayed(_plot_cubed_sphere_panel)(*args) for args in job_args]
+    with Parallel(n_jobs=8, verbose=10) as parallel:
```

Comment:
Do we need to pass n_jobs? or are we okay hard coding it to 8?

Reply:
I think since it's just figure generation and unlikely to be resource hungry, I'm fine with hardcoding.  it was implicitly hardcoded to n_jobs=1 🐢